tests/test_cli.py

```diff
@@ -20,6 +20,13 @@ def test_help_text_no_args(script_runner):
     assert "geoextent [-h]" in ret.stdout, "usage instructions are printed to console"
 
 
+def test_details_folder(script_runner):
```

Comment:
Please add the opposite test as well: by default, details is _not_ in the result.